web/src/types.ts

```diff
@@ -6,13 +6,67 @@ export type MatchedMessage = {
 
 // TODO/typescript: Move this to message_store
 export type MessageType = "private" | "stream";
+export type MessageContentType = "text/html" | "text/x-markdown";
```

Comment:
I don't understand this type -- the message `content` field will be a string, and I think always an HTML string in these contexts, since we always request that format from the server.

Reply:
I inferred this type from reading the [API docs](https://zulip.com/api/get-message), there it is written that:
> The HTTP content_type for the message content. This will be text/html or text/x-markdown, depending on whether 
> apply_markdown was set.

Reply:
Should I just change it to `text/html` only or maybe a `string` type?

Reply:
It should be `"text/html"`. We only get `"text/x-markdown"` for `content_type` when `apply_markdown` is set to `false` upon request, which is not done in the web client.

Reply:
Oh okay makes, changed it to string literal type of "text/html".